### PR TITLE
Review wanted: Christina Persson - Weather app

### PR DESCRIPTION
**Pull request by Cilpersson**
### Link to my deployed project:
Weather app website:
https://tinyweatherapp.netlify.com/

### My reflections on how this week's project turned out:

**What I struggled with and should improve for next weeks project**
I struggled with  knowing where to put styling, there is styling all over the place. In the HTML, CSS (obviously) and Javascript. I'm not at all pleased with my HTML code structure and I feel sorry for the poor soul who is going the read my code. 

**What I'm proud of**
I'm proud of a lot of things. I did some pretty cool stuff this week, the background gradients change with the temperature. The icon not only changes with weather type but it also "breathes". I have also added a night mode, during the night there is a darker gradient and a sleeping full moon.

I'm also proud of my mobile footer, where I used two gradient styled `<hr>`  to make a some what discrete footer.

### CODE REVIEW

**Comment by Cilpersson** [comment]
I know now how to style the backgrounds etc with classes in css and toggle them in javascript. I would have done that differently, but will do that from next week 🤗

**Comment by ericaglimsholt** [comment]
Hi! Is this project done and ready for a review? If yes - please update the PR title to "review wanted:" before your title you have now (you can edit your PR here on github). This is for the projects that's done before the deadline. If you change it to "review wanted" we will try and give you a review as soon as possible.

File: code/index.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        <button id="malmo" onclick="userChoice('Malmoe')">Malm&ouml;</button>
+        <button id="sthlm" onclick="userChoice('Stockholm')">Stockholm</button>
+      </div>
+      <div class="mobile-footer">
+        <hr class="mobile-line" />
+        <div class="icons-mobile">
+          Icons made by
+          <a href="https://www.flaticon.com/authors/freepik" title="Freepik"
+            >Freepik</a
+          >
```
Comment (ericaglimsholt) [inline code/index.html]: I don't know really if this is easier to read in comparence to have everything in one line!

File: code/script.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+      const sunsetTime = timeFormat(json.sys.sunset);
+
+      const weather = json.weather[0].main;
+      const temperature = json.main.temp;
+
+      const currentTime = new Date();
+      const currentHour = currentTime.getHours();
+
+      containerToday.innerHTML = `<h3>${cityFormat(city)}</h3>`;
+      containerToday.innerHTML += `<img src="${checkWeather(
```
Comment (ericaglimsholt) [inline code/script.js]: Ohh like that you have named the images so you can do this!

File: code/script.js [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+  fetch(weatherFiveDays)
+    .then(response => {
+      return response.json();
+    })
+    .then(json => {
+      const filteredForecast = json.list.filter(item =>
+        item.dt_txt.includes("03:00")
+      );
+      console.log(filteredForecast);
```
Comment (ericaglimsholt) [inline code/script.js]:
```suggestion
```

File: code/script.js [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+//Styles background
+const styleContainers = backgroundColour => {
+  containerWeather.style.backgroundImage = backgroundColour;
+  footer.style.background = backgroundColour;
+  header.style.background = backgroundColour;
+};
+
+//Checks temperature and calls styleContainer function to style based on temp
+const tempCheck = temp => {
```
Comment (ericaglimsholt) [inline code/script.js]: I like this!!

File: code/index.html (hunk 1 above)
Comment (Cilpersson) [reply code/index.html]: I don't know why my formatter decided to do this, didn't even notice it. Will have a look in vsc and see if I can change the settings!

File: code/script.js (hunk 4 above)
Comment (Cilpersson) [reply code/script.js]: Thank you for all the kind words on this project! I really tried to put a lot of effort on keeping structure in my js-file with functions for almost everything.  I'm glad that you liked it!

**Pull request closed without merging**